build-logic/kotlin-convention/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+plugins {
+    `kotlin-dsl`
+    id("com.avito.android.libraries")
+}
+
+group = "com.avito.android.buildlogic"
+
+dependencies {
+    implementation("com.avito.android.buildlogic:libraries")
+    implementation(platform("com.avito.android.infra:platform"))
+    implementation(libs.kotlinPlugin)
+    implementation(libs.androidGradlePlugin)
```

Comment:
How about extracting this to "android" convention?
Or renaming `kotlin-convention` to `android-convention`

Reply:
and even more like `conventions` )